stable/kubernetes-dashboard/values.yaml

```diff
@@ -21,6 +21,10 @@ labels: {}
 # extraArgs:
 #   - --enable-insecure-login
 #   - --system-banner="Welcome to Kubernetes"
+deployment:
```

Comment:
Not sure about, either we put every parameter related to the deployment here, or nothing.

Reply:
@desaintmartin how would you suggest we organize the variables?

Reply:
I would suggest directly in the root.